### PR TITLE
Combine GRest meeting summaries, add dbsync commit to version

### PR DESCRIPTION
**Pull request by rdlrt**

### CODE REVIEW

**Comment by rdlrt** [comment]
Question: Should we reverse the order of summaries (loading page shows newest meet summary first and in the content updates are at the top instead of bottom)?

File: docs/Build/grestsvcs.json [hunk 1]
```diff
@@ -413,6 +412,49 @@
           }
         }
       }
+    },
+    "/rpc/rewards_history": {
```
Comment (DamjanOstrelic) [inline docs/Build/grestsvcs.json:416]: Did you mean to include changes to `grestsvcs.json` in this PR?

**Comment by DamjanOstrelic** [comment]
> Question: Should we reverse the order of summaries (loading page shows newest meet summary first and in the content updates are at the top instead of bottom)?

In my opinion, the reverse order is better with the latest ones first.

**Review by DamjanOstrelic — APPROVED** [review]
Great idea to merge them together 👍

File: docs/Build/grestsvcs.json (hunk 1 above)
Comment (rdlrt) [reply docs/Build/grestsvcs.json:416]: ye - can resolve merge conflicts if any on individual PRs, but would be good to get them on frontend soon

**Pull request merged**